config/phenomenal_tools2container.yaml

```diff
@@ -73,7 +73,7 @@ assignment:
   docker_owner_override: phnmnl
   docker_repo_override: container-registry.phenomenal-h2020.eu
   docker_tag_override: v1.1.0_cv1.4.11
-  max_pod_retrials: 3
+  max_pod_retrials: 1
```

Comment:
These retries are useful if the container fails due to resource limitation -> we don't expect failure due to memory, and try to recover with higher limits.

Reply:
The ISA-related tools have very low resource requirements, which is why I reduced these for these tools. If there's something wrong, these kinds of tools should fail quickly I think.

Reply:
I think it's still a good idea to keep a retry number > 1.  Failures from transient conditions, like network problems, will often be solved by a simple retry.